pmm-app/src/pmm-check/CheckPanel.tsx

```diff
@@ -59,6 +63,19 @@ export class CheckPanel extends PureComponent<CheckPanelProps, CheckPanelState>
     }
   }
 
+  async handleRerunChecksClick() {
+    this.setState({ isRerunChecksLoading: true });
```

Comment:
I'd suggest `isRequestPending`, since this is much more resistant to refactoring ))

Reply:
Renamed to `isRunChecksRequestPending`